src/main/webapp/resources/js/components/samples/components/SampleMetadata.tsx

```diff
@@ -173,7 +180,7 @@ export function SampleMetadata() {
             <Empty description={i18n("SampleDetails.no-metadata")} />
           )
         ) : (
-          <ContentLoading />
+          <ContentLoading message={""} props={undefined} />
```

Comment:
This is awful.  If you are not using any of it, then just use the `Spin` component, or update this to handle the use case